src/urbit/app/chess.hoon

```diff
@@ -59,50 +70,81 @@
   ^-  (quip card _this)
   ?+  mark  (on-poke:default mark vase)
     %chess-action
+      ::
+      ::  assert source of the poke is
+      ::  our ship, or one of our moons
       ?>  (team:title our.bowl src.bowl)
       =/  action  !<(chess-action vase)
       ?-  -.action
         %challenge
+          ::
+          ::  if we've already sent this
+          ::  ship an unanswered challenge...
           ?:  (~(has by challenges-sent) who.action)
+            ::  then error!
+            ::  TODO: this should display to frontend
             :_  this
             =/  err
               "already challenged {<who.action>}"
             :~  [%give %poke-ack `~[leaf+err]]
             ==
+          ::  if not, poke them with a %challenge...
           :-  :~  :*  %pass  /poke/challenge  %agent  [who.action %chess]
                       %poke  %chess-challenge  !>(challenge.action)
                   ==
               ==
+          ::  and add them to our list
+          ::  of unanswered challenges
           %=  this
             challenges-sent  (~(put by challenges-sent) +.action)
           ==
```

Comment:
Usually, the goal of comments is to:
- Denote what entire sections of code do
- Save brain computation time by succinctly describing non-trivial operations
- Explain why a particular action is done or style is used in situations where it's not obvious (like sign posts for [Chesterton's Fence](https://thoughtbot.com/blog/chestertons-fence)).

Breathing comments help visually to separate sections from each other, thus usually are used for comment text that applies to an entire block of code. In another language, they would be "block comments" (e.g. `/* ... */`). Comments without them are what other languages would call "in-line" comments (e.g. `//`), which typically serve to explain just the couple of lines following them.

With that in mind, here's how I would have commented this section:
```
?+  mark  (on-poke:default mark vase)
  ::
  ::  pokes managing active game state and challenges
  %chess-action
    ::  only allow chess actions from our ship or our moons
    ?>  (team:title our.bowl src.bowl)
    =/  action  !<(chess-action vase)
    ?-  -.action
      ::
      ::  manage new outgoing challenges
      %challenge
        ::  only allow one active challenge per ship
        ::  TODO: change or display to frontend
        ?:  (~(has by challenges-sent) who.action)
          :_  this
          =/  err
            "already challenged {<who.action>}"
          :~  [%give %poke-ack `~[leaf+err]]
          ==
        ::  send new challenge
        :-  :~  :*  %pass  /poke/challenge  %agent  [who.action %chess]
                    %poke  %chess-challenge  !>(challenge.action)
                ==
            ==
        ::  add to list of outgoing challenges
        %=  this
          challenges-sent  (~(put by challenges-sent) +.action)
        ==
```

This feedback applies for the rest of the file as well